src/frontend/package.json

```diff
@@ -20,53 +22,73 @@
   "author": "France Université Numérique",
   "license": "MIT",
   "devDependencies": {
-    "@babel/core": "7.13.14",
+    "@babel/core": "7.13.10",
```

Comment:
Most of the change you made are downgrading the installed version. Is it a copy pasta fails ?

Reply:
might be a result of a no conflict

Reply:
I'll have to be careful in my next rebase